serverlessworkflow/antora.yml

```diff
@@ -62,7 +62,7 @@ asciidoc:
     podman_install_url: https://docs.podman.io/en/latest/
     kubectl_install_url: https://kubernetes.io/docs/tasks/tools/install-kubectl
     docker_compose_install_url: https://docs.docker.com/compose/install/
-    kn_cli_install_url: https://github.com/knative/client/blob/main/docs/README.md#installing-kn
+    kn_cli_url: https://knative.dev/docs/client/install-kn/
```

Comment:
I would keep `kn_cli_install_url` as the attribute name

Reply:
okay